picard/ui/widgets/scripttextedit.py

```diff
@@ -125,16 +125,15 @@ class TaggerScriptSyntaxHighlighter(QtGui.QSyntaxHighlighter):
 
     def __init__(self, document):
         super().__init__(document)
-        syntax_theme = theme.syntax_theme
 
         self.textcharformats = {
-            'escape': HighlightFormat(fg_color=syntax_theme.escape),
-            'func': HighlightFormat(fg_color=syntax_theme.func, bold=True),
-            'noop': HighlightFormat(fg_color=syntax_theme.noop, bold=True, italic=True),
-            'special': HighlightFormat(fg_color=syntax_theme.special),
-            'unicode': HighlightFormat(fg_color=syntax_theme.escape, italic=True),
-            'unknown_func': HighlightFormat(fg_color=syntax_theme.special, italic=True),
-            'var': HighlightFormat(fg_color=syntax_theme.var),
+            'escape': HighlightFormat(fg_color='syntax_hl_escape'),
+            'func': HighlightFormat(fg_color='syntax_hl_func', bold=True),
+            'noop': HighlightFormat(fg_color='syntax_hl_noop', bold=True, italic=True),
+            'special': HighlightFormat(fg_color='syntax_hl_special'),
+            'unicode': HighlightFormat(fg_color='syntax_hl_unicode', italic=True),
```

Comment:
Minor thing I noticed during testing: The unicode escapes have become italic in #2492 . This is looking odd, especially due to the backslash getting tilted. Maybe you can change this before merging this PR, or we change it after merging.

Reply:
I set it to italic because it was using the same color as 'escape', but now we can have different colors so I guess we can remove it.
I'd better adjust overall style after we merged this, we can define slightly different colors for each case.